py/sight/utils/.text_proto_configs/calculator.textproto

```diff
@@ -0,0 +1,26 @@
+action_attrs {
```

Comment:
move these textproto files into the same directory as the worker yml files for these workers. It will be easier to keep all the config files related to a given worker in the same place